DotNET/Endpoint Examples/JSON Payload/decrypted-pdf.cs

```diff
@@ -46,6 +49,31 @@
 
             Console.WriteLine("Processing response received.");
             Console.WriteLine(decryptResult);
+
+            // All files uploaded or generated are automatically deleted based on the 
+            // File Retention Period as shown on https://pdfrest.com/pricing. 
+            // For immediate deletion of files, particularly when sensitive data 
+            // is involved, an explicit delete call can be made to the API.
+            //
+            // The following code is an optional step to delete sensitive files 
+            // (unredacted, unencrypted, unrestricted, or unwatermarked) from pdfRest servers.
```

Comment:
I really think we should delete **all** of the files partly just to show that we can and partly since I think people who would be interested in it might still want their files to be deleted even when they are watermarked/encrypted/redacted etc